src/app/community/community.component.ts

```diff
@@ -36,13 +36,13 @@ export class CommunityComponent implements OnInit, AfterViewInit {
 
   ngOnInit() {
     this.getCommunityList();
-    this.communities.sortingDataAccessor = (item, property) => item[property].toLowerCase();
+    this.communities.sort = this.sort;
```

Comment:
Rather than reset the sort function to be case sensitive, you should do a type check on the `item[property]` and only to the `toLowerCase()` transformation if it is a string.

Reply:
No, we need to use toLowerCase() when it is a string.  Otherwise it will sort uppercase letters first, then lowercase next.